components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -48,15 +48,14 @@ end
 ---@return table[]
 function MatchFunctions.extractMaps(match, opponents)
 	local maps = {}
-	for key, map, mapIndex in Table.iter.pairsByPrefix(match, 'map', {requireIndex = true}) do
+	for key, map in Table.iter.pairsByPrefix(match, 'map', {requireIndex = true}) do
 		if map.map == nil then
 			break
 		end
 		local finishedInput = map.finished --[[@as string?]]
 		local winnerInput = map.winner --[[@as string?]]
 
 		map.extradata = MapFunctions.getExtraData(map)
-		map.vod = map.vod or String.nilIfEmpty(match['vodgame' .. mapIndex])
```

Comment:
same

Reply:
Yes, Kano removed the few usages of this yday (there were like 15)